### PR TITLE
*/* EAPI 7 sci@ coinor-* autotools-utils removal batch

### PR DESCRIPTION
**Pull request by ionenwks**
Also 2 others on top and mal.. oh nevermind :)

Also included alps to be in line with bcps/blis (alps is ltprune/eutils).

### CODE REVIEW

File: sci-libs/mc/mc-1.5.ebuild [hunk 1]
```diff
@@ -26,59 +25,38 @@ RDEPEND="
...
 }
 
 src_configure() {
-	local fetk_include
-	local fetk_lib
-	local myeconfargs
-
-	use doc || myeconfargs+=( --with-doxygen= --with-dot= )
-
-	fetk_include="${EPREFIX}"/usr/include
```
Comment (thesamesam) [inline sci-libs/mc/mc-1.5.ebuild:62]: Huh, that's a huge chunk gone? Wasn't needed at all?

(Sure it's not falling back on anything bundled?)

Comment (ionenwks) [reply sci-libs/mc/mc-1.5.ebuild:62]: I did compare shared libraries used, and was still same amount. I did start to doubt when I moved on to punc given things were different there.

Comment (ionenwks) [reply sci-libs/mc/mc-1.5.ebuild:62]: It's also same size so I'd assume nothing got added, but I admit the configure messages worry me. Maybe worth more looking into.

**Comment by thesamesam** [comment]
Pushed all but gamer, mc. let's see what @SoapGentoo says re last-riting it and punc et al. or not.

File: sci-libs/coinor-cppad/coinor-cppad-20210000.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+DESCRIPTION="COIN-OR C++ Algorithmic Differentiation"
+HOMEPAGE="https://projects.coin-or.org/CppAD/"
+SRC_URI="https://github.com/coin-or/CppAD/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/CppAD-${PV}"
+
+LICENSE="EPL-2.0"
+SLOT="0/${PV}" # soname is bumped every versions
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="adolc doc eigen ipopt"
+
```
Comment (SoapGentoo) [inline sci-libs/coinor-cppad/coinor-cppad-20210000.5.ebuild:18]: remvoe empty line (`RESTRICT` is part of this block)

File: sci-libs/coinor-symphony/coinor-symphony-5.6.17.ebuild [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+src_configure() {
+	econf $(usex glpk --with-glpk-lib=-lglpk --without-glpk)
+}
+
+src_compile() {
+	default
+	if use doc; then
+		pushd Doc && pdflatex Walkthrough && pdflatex man && popd || die
```
Comment (SoapGentoo) [inline sci-libs/coinor-symphony/coinor-symphony-5.6.17.ebuild:51]: separate this over multiple lines, a bit too terse

File: sci-libs/coinor-symphony/coinor-symphony-5.6.17.ebuild [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+	emake test
+}
+
+src_install() {
+	default
+	use doc && dodoc Doc/*.pdf
+
+	# Other coinor-* use lowercase e, stay in-line with them.
+	docinto examples
+	dodoc -r Examples/*
```
Comment (SoapGentoo) [inline sci-libs/coinor-symphony/coinor-symphony-5.6.17.ebuild:66]: `dodoc -r Examples/.` (avoid globs)

**Review by SoapGentoo — CHANGES_REQUESTED** [review]
@ionenwks well done, this is a great chunk of work!

**Comment by ionenwks** [comment]
Thanks for review :)

Since coinor-* is merged, and I'd rather not handle fetk packages (mc/gamer) I think we're done here, closing.

**Pull request closed without merging**